ui/css/perf.css

```diff
@@ -455,3 +455,29 @@ td.alert-confidence {
 .alert-strike {
     text-decoration: line-through;
 }
+
+#loading-symbol {
+  position: relative; /* So we can absolutely position the loading overlay */
+}
+.overlay {
```

Comment:
You should have a blank line between the end of each css directive and the beginning of a new one.  Example:

```css
#loading-symbol {
  ...
}

.overlay {
```